pkg/autoscaler/autoscaler.go

```diff
@@ -142,16 +156,12 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (desiredPodCount
 	desiredStablePodCount := int32(math.Min(math.Ceil(observedStableValue/spec.TargetValue), maxScaleUp))
 	desiredPanicPodCount := int32(math.Min(math.Ceil(observedPanicValue/spec.TargetValue), maxScaleUp))
 
-	a.reporter.ReportStableRequestConcurrency(observedStableValue)
-	a.reporter.ReportPanicRequestConcurrency(observedPanicValue)
-	a.reporter.ReportTargetRequestConcurrency(spec.TargetValue)
-
-	logger.Debugw(fmt.Sprintf("Observed average %0.3f concurrency, targeting %0.3f.",
-		observedStableValue, spec.TargetValue),
-		zap.String("concurrency", "stable"))
-	logger.Debugw(fmt.Sprintf("Observed average %0.3f concurrency, targeting %0.3f.",
-		observedPanicValue, spec.TargetValue),
-		zap.String("concurrency", "panic"))
+	logger.Debugw(fmt.Sprintf("Observed average %0.3f %v, targeting %0.3f.",
+		observedStableValue, metricName, spec.TargetValue),
```

Comment:
metric name is string, so %s, but why not just annotate the logger with it above?

Reply:
I took markusthoemmes's suggestion. I think having the metric in the sentence is more readable.